app/test_main.py

```diff
@@ -1,3 +1,37 @@
+import pytest
+
+from typing import Any
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        pytest.param(-1, -2, [0, 0]),
+        pytest.param(0, 0, [0, 0],),
+        pytest.param(14, 14, [0, 0],),
+        pytest.param(15, 15, [1, 1],),
+        pytest.param(23, 23, [1, 1],),
+        pytest.param(24, 24, [2, 2],),
+        pytest.param(27, 28, [2, 2],),
+        pytest.param(28, 29, [3, 3],),
+        pytest.param(100, 100, [21, 17],),
+    ]
+)
+def test_correct_human_years(cat_age: int,
```

Comment:
after the opening bracket, write all arguments on a new line, including the first one

Reply:
Done